examples/SRParallelArchipelagoExample.py

```diff
@@ -71,9 +71,10 @@ def execute_generational_steps():
 
     opt_result = archipelago.evolve_until_convergence(max_generations=500,
                                                       fitness_threshold=1.0e-4)
+    best_indv = archipelago.get_best_individual()
```

Comment:
Need all ranks to get the best individual, so it can't be under `rank == 0` or the program hangs when run with mpi. Updated `.test_examples.sh` to test for this.